using_images/other_images/other_container_images.adoc

```diff
@@ -0,0 +1,48 @@
+[[using-images-other-container-images]]
+= Other Container Images
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+== Overview
+
+If you want to use container images not found in the
+link:https://access.redhat.com/containers/[Red Hat Container Catalog], you can
+use other arbitrary container images in your {product-title} instance, for
+example those found on the https://registry.hub.docker.com/[Docker Hub].
+
+For more information on support for images running as any arbitrary user ID, see
+xref:../../creating_images/guidelines.adoc#openshift-specific-guidelines[{product-title}-Specific
+Guidelines] in the Creating Images guide.
```

Comment:
there's no anchor, but you can direct them specifically to the "Support Arbitrary User IDs" section of that doc.

Reply:
Donezo.